compiler/rustc_target/src/spec/mod.rs

```diff
@@ -122,6 +123,7 @@ impl ToJson for LldFlavor {
             LldFlavor::Ld => "gnu",
             LldFlavor::Link => "link",
             LldFlavor::Wasm => "wasm",
+            LldFlavor::Gcc => todo!(),
```

Comment:
```suggestion
            LldFlavor::Gcc => "gcc",
```
and also add it to the `from_str` impl above.